WooCommerce/Classes/ViewModels/Order Details/OrderDetailsResultsControllers.swift

```diff
@@ -124,10 +132,21 @@ private extension OrderDetailsResultsControllers {
             onReload()
         }
 
-        refundResultsController.onDidResetContent = {
+        refundResultsController.onDidResetContent = { [weak self] in
+            guard let self = self else {
+                return
+            }
+            self.refetchAllResultsControllers()
             onReload()
         }
 
         try? refundResultsController.performFetch()
     }
+
```

Comment:
I'd say to add a small comment about why we are doing these refetch. 
To the careless eye, it may seem superfluous and may be accidentally removed in the future.

Reply:
yes good idea! it does help with anyone reading the code in the future, added in https://github.com/woocommerce/woocommerce-ios/pull/2821/commits/f75f00336f8133fe704b16b0d33acee749387168